Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM runatlantis/atlantis:latest
+
+ENV TERRAGRUNT_VERSION=v0.21.7
```

Comment:
Another question: as Terragrunt updates quite frequently, how are we going to catch up with that (other than PR-ing to this repo every week)? Or should we rather always use the latest version?

Reply:
The what about using the latest Terragrunt version instead?
@czerasz-flaconi @maartenvanderhoef